lighthouse-core/report/html/renderer/report-renderer.js

```diff
@@ -223,20 +223,23 @@ class ReportRenderer {
     // }
 
     if (scoreHeader) {
-      const defaultGauges = [];
-      const customGauges = [];
+      // Group gauges in this order: mainstream, pwa, plugins.
+      const mainstreamGauges = [];
```

Comment:
can we still use "deault" (from `default-config.js`) or "core" for these?

Reply:
done